service/controller/app/v1/resource/clients/resource.go

```diff
@@ -125,9 +126,10 @@ func (r *Resource) addClientsToContext(ctx context.Context, cr v1alpha1.App) err
 			K8sClient: cc.K8sClient,
 			Logger:    r.logger,
 
-			RestConfig:          restConfig,
-			TillerImageRegistry: r.imageRegistry,
-			TillerNamespace:     r.tillerNamespace,
+			EnsureTillerInstalledMaxWait: 30 * time.Second,
```

Comment:
Reduces the max wait from 2 mins which is the default. This is needed for integration tests but is far too long here.